Add example with vertical bars in BarSeriesExamples.cs

mennowo opened the pull request:
In the past, there was ColumnSeries, which could have the categories be placed on the X axis. The BarSeries explicitly needs them on the Y axis. The new example show how use `XAxisKey` and `YAxisKey` to have categories on bottom axis in the plot. There is no documentation for this, so it would be good example to have out there.

### Checklist

- [X] I have included examples or tests
- [X] I have updated the change log
- [X] I am listed in the CONTRIBUTORS file
- [X] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

- Add example for `BarSeries`, showing how to make it plot vertical bars

@oxyplot/admins

mennowo committed:
Add example with vertical bars in BarSeriesExamples.cs

In the past, there was ColumnSeries, which could have the categories be placed on the X axis. The BarSeries needs them on the Y axis. The new example show how to do this, but use keys to have to categories on bottom axis in the plot anyway.

Update CONTRIBUTORS

Update CHANGELOG.md

mennowo commented on CONTRIBUTORS:
```diff
@@ -154,4 +155,4 @@ R. Usamentiaga
 Konstantin Stukov <havendv@gmail.com>
-jorgectf
\ No newline at end of file
+jorgectf
```
sorry about this, I don't know how this got in here

VisualMelon replied:
Your editor probably put a new line at the end on saving: it's not a problem.

mennowo commented:
See also #1870 where I found the info which lead me to experiment and finally found out that vertical bars were indeed: possible!

VisualMelon commented on CHANGELOG.md:
```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [2.2.1] - 2024-12-11
```
Should be Unreleased (my problem)

VisualMelon commented:
This is a very good idea: it is indeed a question that is asked a lot.

Probably better to have a 'full' example for this, which doesn't require looking into other methods and updating the axes later. Something close to https://github.com/oxyplot/oxyplot/discussions/1946#discussioncomment-3806771 perhaps

mennowo commented:
Excellent, shall I update the PR to implement a full example? 

I did not find the discussion you referenced, that would have been helpful :-) But, I only searched via google and in the issues, and not in the discussions.

VisualMelon commented:
Go for it: I should have time to review it properly this weekend (feel free to ping me if I don't reply)

mennowo committed:
#2116 add full example for vertical bar series

mennowo commented:
I now added this full example. Is it possible to merge this as is? Or do I need to do some kind of rebase, cause it has been while since I opened this PR (sorry...) ?

The example is now based on the first bar series example, where I removed labels and added verticality; quite a clear case.

VisualMelon commented:
Looks fine to merge to me: I'll check it properly later today and try get it merged sooner rather than later. As always, feel free to ping me if I don't get around to it.

Rules on rebasing are mostly to make sure whoever submits the PR is on a sensibly recent commit so that it doesn't produce a nightmarish merge request: no conflicts here and it's all new and self-contained code so nothing really to go wrong,

mennowo commented:
@VisualMelon Have had a chance to review this? I'm happy to adjust as needed. Afterwards, I can then create a PR to allow manually loading fonts in a WASM context. Thank you!

VisualMelon commented:
Yep, looks good to me.

If it's no effort, could you change the release notes heading to "Unreleased" (rather than "[2.2.1] - 2024-12-11")? (I should have done that ages ago)

mennowo committed:
#2116 Alter version comment for most recent changes

mennowo commented:
Ok, I now changed the most recent version name in the changelog.
